runtimes/alibi-detect/mlserver_alibi_detect/cd/runtime.py

```diff
@@ -0,0 +1,42 @@
+from mlserver_alibi_detect.runtime import AlibiDetectRuntime
+from mlserver_alibi_detect.errors import InvalidAlibiDetector
+from mlserver.utils import get_model_uri
+from alibi_detect.utils.saving import load_detector
+from importlib import import_module
+from typing import Any
+import pickle
+
+ALIBI_DETECT_DRIFT_MODULE = "alibi_detect.cd"
+
+
+class AlibiDriftDetectRuntime(AlibiDetectRuntime):
+    """
+    Implementation of the MLModel interface to load and serve Alibi drift models.
+    """
+
+    async def load(self) -> bool:
+
+        model_uri = await get_model_uri(self._settings)
+
+        if not bool(self.alibi_detect_settings.init_detector):
```

Comment:
This one should always be a boolean now, right? In that case, we could get rid of the explicit `bool()` casting.

Reply:
Yeah will update